frontend/src/components/configurate-episode/components/create-episode-form/components/record-preview-control/record-preview-control.tsx

```diff
@@ -28,16 +30,22 @@ const RecordPreviewControl: React.FC<Props> = ({
   name,
   control,
   errors,
+  hasLiveRecord,
 }) => {
   const { field } = useController({ name, control });
 
   const [currentRecord, setRecord] = useState<string>('');
 
+  const liveRecordName = hasLiveRecord
+    ? `${getFormattedDate(new Date(), DateFormatType.DAY_MONTH_YEAR)}.${FileExtension.WAV}`
+    : '';
```

Comment:
create a `getLiveRecorName` helper and move it to frontend/src/components/configurate-episode/components/create-episode-form/components/record-preview-control/helpers/